src/settings.c

```diff
@@ -41,6 +41,14 @@ static void on_sidebar_pos_left_changed(GSettings *settings, gchar *key, gpointe
 }
 
 
+static void on_editor_font_changed(GSettings *settings, gchar *key, gpointer user_data)
+{
```

Comment:
I was hoping that there was enough similarity between them to maybe have a few callbacks and pass sufficient information via the `user_data`.  For example this type of setting would handle all string settings that call a set function, and the user data would need a pointer to the setting of course, and a pointer to a function to call (set to `ui_set_editor_font()` for this setting).